src/unit-taxonomy-tags-drawer/TagBubble.test.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+import { initializeMockApp } from '@edx/frontend-platform';
+import { AppProvider } from '@edx/frontend-platform/react';
+import { render } from '@testing-library/react';
+import PropTypes from 'prop-types';
+
+import initializeStore from '../store';
+
+import TagBubble from './TagBubble';
+
+let store;
+
+const data = {
+  value: 'Tag 1',
+};
+
+const TagBubbleComponent = ({ value, subTagsCount, implicit }) => (
+  <AppProvider store={store}>
```

Comment:
Since we are using React Query instead of redux for this stuff, you actually don't need `<AppProvider>` here in your test suite at all :)